bin/latex.py

```diff
@@ -209,38 +259,38 @@ def build_problem_pdfs(problem, solutions=False, web=False):
     if config.args.languages is not None:
         for lang in config.args.languages:
             if lang not in problem.statement_languages:
-                fatal(f"No statement source for language {lang}")
+                fatal(f'No statement source for language {lang}')
         languages = config.args.languages
     else:
         languages = problem.statement_languages
         # For solutions, filter for `solution.xy.tex` files that exist.
         if solutions:
             filtered_languages = []
             for lang in languages:
-                if (problem.path / f"problem_statement/solution.{lang}.tex").exists():
+                if (problem.path / 'problem_statement' / f'solution.{lang}.tex').exists():
                     filtered_languages.append(lang)
                 else:
                     warn(f'{problem.name}: solution.{lang}.tex not found')
             languages = filtered_languages
-
-    return all(build_problem_pdf(problem, lang, solutions, web) for lang in languages)
+    if config.args.watch and len(languages) > 1:
+        fatal('--watch does not work with multiple languages. Please use --language')
+    return all([build_problem_pdf(problem, lang, solutions, web) for lang in languages])
```

Comment:
do the `[]` here do anything?

Reply:
they disable short circuit evaluation i.e. dont stop after the first pdf is broken